3ds/get_pid_by_lfc.go

```diff
@@ -0,0 +1,29 @@
+package friends_3ds
+
+import (
+	"github.com/PretendoNetwork/friends-secure/globals"
+	nex "github.com/PretendoNetwork/nex-go"
+	nexproto "github.com/PretendoNetwork/nex-protocols-go"
+)
+
+func GetPrincipalIDByLocalFriendCode(err error, client *nex.Client, callID uint32, lfc uint64, lfcList []uint64) {
+	// Drop this request entirely, waiting for gRPC to retrieve PID from account server
+
+	rmcResponse := nex.NewRMCResponse(nexproto.Friends3DSProtocolID, callID)
+	rmcResponse.SetSuccess(nexproto.Friends3DSMethodGetPrincipalIDByLocalFriendCode, nil)
```

Comment:
This should probably be an RMC error using the `Unimplemented` error code, unless the success is required for something else?